src/component/number.js

```diff
@@ -1,10 +1,14 @@
-const Num = () => (
+/* eslint-disable import/no-extraneous-dependencies */
+import PropTypes from 'prop-types';
```

Comment:
- [ ] Kindly see to it that no linters is disabled, I think you can workaround it without disabling it; this [turorial](https://laurieontech.com/posts/eslint/) might be of help. **While disabling a rule may be necessary in some cases, it's generally recommended to follow the linting rules to maintain code consistency and prevent potential issues.**